certif/tests/unit/controllers/authenticated/sessions/details/certification-candidates_test.js

```diff
@@ -38,17 +38,18 @@ module('Unit | Controller | authenticated/sessions/details/certification-candida
     });
   });
 
-  module('#get shouldDisplayComplementaryCertifications', function () {
+  module('#get shouldDisplayComplementaryCertificationsHabilitations', function () {
     test('should return false if center has no complementary certifications', function (assert) {
```

Comment:
should return false if center has no complementary certifications habilitations ?